### ISSUE TITLE
Feature Request Sort by Artist

### ISSUE THREAD

**Issue by libasys** [libasys/audios#1]
The possibilty to sort songs by artist, year, etc!
##

**Comment by Atanamo** [comment]
Does "sorting" only mean to sort the existing albums view?

It would be a bigger benefit to have a kind of filter or additional kinds of views.
If your have a huge music library (100+ albums), you don't want to see all albums in one list - it is too much information and much too slow anyway.

What about an artists view? 
It basically works like the current albums view, but lists only artists. When you select an artist, the list of all songs of the artist appears - PLUS a link to the albums view:
When you click the link, the current albums view is loaded, but with a filter on the artist's albums.

By this one could easily navigate through a huge library...

Greetings from Hessen ;-)

**Comment by Aidenir** [comment]
I agree with @Atanamo. I personally have no use whatsoever of the album view, since none of the files I have are even a part of any album, mostly mixes from youtube. However all of them have the artist field filled in. Would love to see an Artist view, and preferably just the simplest one, a tracks view, which just shows all tracks in a list.
